src/sequelize/managers/kubelet-access-token-manager.js

```diff
@@ -0,0 +1,36 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2018 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+const BaseManager = require('./base-manager');
+const models = require('./../models');
+const FogAccessToken = models.FogAccessToken;
+
+class KubeletAccessTokenManager extends BaseManager {
+  getEntity() {
+    return FogAccessToken
+  }
+
+  // no transaction required here, used by auth decorator
+  updateExpirationTime(id, newTime) {
```

Comment:
i think, that transaction should be here, because method called from service functions, not from decorators

Reply:
fix